reg_form_ext/models.py

```diff
@@ -0,0 +1,23 @@
+from django.conf import settings
+from django.db import models
+from .choices import COUNTRY_CODES
+
+# Backwards compatible settings.AUTH_USER_MODEL
+USER_MODEL = getattr(settings, 'AUTH_USER_MODEL', 'auth.User')
+
+
+class PhoneInfo(models.Model):
+    """
+    Holds two fields for the City and Country.
+    Used during user registration as a form extension.
+    """
+    user = models.OneToOneField(USER_MODEL, null=True)
+    country_code = models.CharField(
+        verbose_name="Country Calling Code",
+        max_length=7,
+        choices=COUNTRY_CODES
+    )
+    phone_number = models.CharField(
+        verbose_name="Phone Number",
+        max_length=60
+    )
```

Comment:
[As noted](https://github.com/open-craft/edx-custom-registration-form/pull/1#issuecomment-348395273), if you use [django-phonenumber-field](https://github.com/stefanfoulis/django-phonenumber-field), you can store these in a single field.